### ISSUE TITLE
[$25]Fix react-ui dependency and variable naming for Accessibility changes 

### ISSUE THREAD

**Issue by codeMinter** [topcoder-platform/community-app#3060]
During the Accessibility we have used some variables in `react-ui` which are not in consistency with https://marvelapp.com/c7h2a6h/screen/59648056/handoff

Once the Accessibility Bug Bash is over, we will need to align this in `react-ui` and `community-app` repo to make them consistent and aligned to our handoff. 

@veshu @sushilshinde @mishacucicea tagging you all so we don't miss this once we wind up Bug Bash.

**Comment by codeMinter** [comment]
Fix this too - https://github.com/topcoder-platform/community-app/issues/2797#issuecomment-513490938

`Use the latest green color from GUI KIT.`

**Comment by veshu** [comment]
We will fix this as the last issue

**Comment by veshu** [comment]
@topcoder-platform/topcodercompetitors This is open for pickup.

The fix is to remove the color variable not present on the handoff from https://github.com/topcoder-platform/topcoder-react-ui-kit and replace those variables used on the https://github.com/topcoder-platform/community-app and https://github.com/appirio-tech/accounts-app with respective allowed color variable.

e.g. $tc-dark-blue-120: #006ad7; in https://github.com/topcoder-platform/topcoder-react-ui-kit/blob/qa-accessibility/src/styles/_mixins/variables.scss is not allowed as handoff doesn't describe `tc-dark-blue-120` so we need to remove this variable and replace this with `tc-dark-blue-110` in community-app and accounts-app if that is used.

Another color variable to replace is '$tc-green-120: #328732;` and use `$tc-green-110: #328732`

Please make sure there is no other discrepancies on handoff guide and color variables.

Let us know if you need more info.

**Comment by r0hit-gupta** [comment]
@veshu as per the marvel handoff, the variable `$tc-green-110` does not equals `#328732`. Can you please confirm what color to use? Thanks!

**Comment by veshu** [comment]
> @veshu as per the marvel handoff, the variable `$tc-green-110` does not equals `#328732`. Can you please confirm what color to use? Thanks!

@Dara-K Can you please help here? The color you have provided 328732 is not present on handoff. Whether we should update the palette or have $tc-green-120 variable as of now. Thanks!

**Comment by Dara-K** [comment]
That's a really good question. When I updated the GUI KIT, I used the #258205 very dark green color that works for texts contrasts and passes accessibility test level AA2. Later, For the graphics elements, I tried to do a lighter version of green - #328732 which would still pass our accessibility tests, but not be that dark, as the colors are losing their contrasts between them and vibrance by darkening them so much, but I didn't add that to the GUI KIT.

To answer your question, based on the lightness of color, it should be:
tc-green-120: #258205
tc-green-110:  #328732
But I am afraid this might mess with the other colors updates we did.

**Comment by r0hit-gupta** [comment]
@veshu how should we proceed here?

**Comment by veshu** [comment]
@r0hit-gupta Per @Dara-K reply, we need to interchange the color code of 120 and 110. instead of removing 120 from ui-kit.

And replace the usage of `tc-green-120` with `tc-green-110` and vice-versa. 

Let me know if it will break the work we did or any concern. Thanks!

**Comment by r0hit-gupta** [comment]
PRs at https://github.com/topcoder-platform/topcoder-react-ui-kit/pull/26 & https://github.com/topcoder-platform/community-app/pull/3272

**Comment by Dara-K** [comment]
@veshu I am not sure how you use the variables in coding, but I recall the current, tc-green-110, was used for color handles and other green texts like TCO small tag/challenge type, etc.  If we update tc-green-110 to #328732, it won't pass the accessibility test so we will probably have to switch variables? just my 2 cents - I will leave it up to you.

**Comment by veshu** [comment]
@Dara-K we are inter changing the values, so it won't affect the previous work we did, it just matching the variable with color shade you said above. @r0hit-gupta can you please confirm my understanding won't break the accessibility?

**Comment by r0hit-gupta** [comment]
@veshu you are right, interchanging the values shouldn't break anything.

**Comment by lakshmiathreya** [comment]
@veshu @r0hit-gupta @sushilshinde pls clarify how is this fix to be reviewed in QA?

**Comment by lakshmiathreya** [comment]
@drasticdpk @tosha5252 fyi

**Comment by veshu** [comment]
@lakshmiathreya @drasticdpk @tosha5252 For this please verify that the accessibility is not broken for green color elements

**Comment by drasticdpk** [comment]
Hi @veshu  @r0hit-gupta , here is few things which is break due to above changes , please update accordingly.
![image](https://user-images.githubusercontent.com/25297859/64330340-7bf0f380-cff0-11e9-9c1d-dd8c02f1d5e3.png)

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30100341<br/><br/>```This is an automated message for crazyk via Topcoder X```
